bootstrap/spigot/src/main/java/org/geysermc/geyser/platform/spigot/GeyserSpigotPlugin.java

```diff
@@ -165,8 +163,31 @@ public void onEnable() {
             return;
         }
 
-        this.geyserCommandManager = new GeyserSpigotCommandManager(geyser);
-        this.geyserCommandManager.init();
+        var sourceConverter = new CommandSourceConverter<>(CommandSender.class, Bukkit::getPlayer, Bukkit::getConsoleSender);
+        PaperCommandManager<GeyserCommandSource> cloud;
+        try {
+            cloud = new PaperCommandManager<>(
```

Comment:
just curious, what's the reason that we're using the PaperCommandManager instead of the BukkitCommandManager? Or is the paper one fully backwards compatible with spigot?

Reply:
yes: <https://github.com/Incendo/cloud/tree/master/cloud-minecraft#cloud-paper>

i thought i had a comment about this, but i must have forgotten it when copying code from crossplatforms